<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="previewPage">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Sprite Editor</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="deleteFrameButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>760</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="framePerSecLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>690</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frames/Second: 1 fps</string>
    </property>
   </widget>
   <widget class="QListWidget" name="frameList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="fpsSlider">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>720</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicateButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>720</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>680</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>600</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>791</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>791</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="toolBoxFrame">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>111</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="eraserButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Eraser</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentColorLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Tool Color:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="currentColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 7px; background-color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Current Color</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolColorButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Tool Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inverseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Inverse Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="text">
      <string>Brush Tool</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brushSizeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Brush Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mirrorBrush">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Mirror Brush</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearFrameButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Frame</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="toolsForFrameFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="widthLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="setXPaintArea">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="heightLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="setYPaintArea">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="setPaintArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Frame Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="flipHorizontal">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>250</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip Frame Horizontal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="flipVertical">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>210</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip Frame Vertical</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="previewPanel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>610</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <zorder>toolBoxFrame</zorder>
   <zorder>previewPage</zorder>
   <zorder>scrollArea</zorder>
   <zorder>deleteFrameButton</zorder>
   <zorder>framePerSecLabel</zorder>
   <zorder>frameList</zorder>
   <zorder>fpsSlider</zorder>
   <zorder>duplicateButton</zorder>
   <zorder>addFrameButton</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>toolsForFrameFrame</zorder>
   <zorder>previewPanel</zorder>
   <zorder>undoButton</zorder>
   <zorder>redoButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInverse"/>
    <addaction name="actionFlip_Horizontal"/>
    <addaction name="actionFlip_Vertical"/>
    <addaction name="separator"/>
    <addaction name="actionClearFrame"/>
   </widget>
   <widget class="QMenu" name="menuBrush">
    <property name="title">
     <string>Brush</string>
    </property>
    <addaction name="actionSize"/>
    <addaction name="actionColor"/>
    <addaction name="actionMirror"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Sprite"/>
    <addaction name="actionSave_Sprite"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
   <addaction name="menuBrush"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionInverse">
   <property name="text">
    <string>Inverse Frame Color</string>
   </property>
  </action>
  <action name="actionFlip_Horizontal">
   <property name="text">
    <string>Flip Frame Horizontal</string>
   </property>
  </action>
  <action name="actionFlip_Vertical">
   <property name="text">
    <string>Flip Frame Vertical</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionClearFrame">
   <property name="text">
    <string>Clear Frame</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="actionMirror">
   <property name="text">
    <string>Mirror</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Sprite">
   <property name="text">
    <string>Open Sprite</string>
   </property>
  </action>
  <action name="actionNew_Sprite">
   <property name="text">
    <string>New Sprite</string>
   </property>
  </action>
  <action name="actionSave_Sprite">
   <property name="text">
    <string>Save Sprite</string>
   </property>
  </action>
  <action name="actionView_Preview">
   <property name="text">
    <string>Preview Sprite</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
